components/ThankYou.js

```diff
@@ -6,6 +6,7 @@ export const ThankYou = () => {
       <h1 className='text-4xl my-3 sm:m-3 sm:block font-bold text-center dark:text-white'>
         Your invite is on its way 🥳
       </h1>
+      <p className={'text-center dark:text-white mb-10'}>Text message sent to the phone number provided</p>
```

Comment:
I wanted to add this text right below, just like the "(iPhone only)" text in the previous state.